<<pipenv>> <<1:5-11>> <<module>> <<VARIABLE>> <<ANCHOR>>
<<patched>> <<1:12-19>> <<module>> <<VARIABLE>> <<ANCHOR>>
<<safety>> <<1:20-26>> <<module>> <<VARIABLE>> <<ANCHOR>>
<<constants>> <<1:27-36>> <<module>> <<VARIABLE>> <<ANCHOR>>
<<SafetyException>> <<5:249-264>> <<type>> <<CLASS>> <<ANCHOR>>
<<__init__>> <<7:286-294>> <<function>> <<CONSTRUCTOR>> <<ANCHOR>>
<<get_exit_code>> <<11:456-469>> <<function>> <<METHOD>> <<ANCHOR>>
<<SafetyError>> <<15:518-529>> <<type>> <<CLASS>> <<ANCHOR>>
<<__init__>> <<17:551-559>> <<function>> <<CONSTRUCTOR>> <<ANCHOR>>
<<get_exit_code>> <<21:688-701>> <<function>> <<METHOD>> <<ANCHOR>>
<<MalformedDatabase>> <<25:750-767>> <<type>> <<CLASS>> <<ANCHOR>>
<<__init__>> <<27:791-799>> <<function>> <<CONSTRUCTOR>> <<ANCHOR>>
<<get_exit_code>> <<34:1212-1225>> <<function>> <<METHOD>> <<ANCHOR>>
<<DatabaseFetchError>> <<38:1279-1297>> <<type>> <<CLASS>> <<ANCHOR>>
<<__init__>> <<40:1321-1329>> <<function>> <<CONSTRUCTOR>> <<ANCHOR>>
<<get_exit_code>> <<44:1465-1478>> <<function>> <<METHOD>> <<ANCHOR>>
<<DatabaseFileNotFoundError>> <<48:1552-1577>> <<type>> <<CLASS>> <<ANCHOR>>
<<__init__>> <<50:1608-1616>> <<function>> <<CONSTRUCTOR>> <<ANCHOR>>
<<get_exit_code>> <<55:1804-1817>> <<function>> <<METHOD>> <<ANCHOR>>
<<InvalidKeyError>> <<59:1896-1911>> <<type>> <<CLASS>> <<ANCHOR>>
<<__init__>> <<61:1942-1950>> <<function>> <<CONSTRUCTOR>> <<ANCHOR>>
<<get_exit_code>> <<69:2335-2348>> <<function>> <<METHOD>> <<ANCHOR>>
<<TooManyRequestsError>> <<73:2405-2425>> <<type>> <<CLASS>> <<ANCHOR>>
<<__init__>> <<75:2456-2464>> <<function>> <<CONSTRUCTOR>> <<ANCHOR>>
<<get_exit_code>> <<81:2674-2687>> <<function>> <<METHOD>> <<ANCHOR>>
<<NetworkConnectionError>> <<85:2746-2768>> <<type>> <<CLASS>> <<ANCHOR>>
<<__init__>> <<87:2799-2807>> <<function>> <<CONSTRUCTOR>> <<ANCHOR>>
<<RequestTimeoutError>> <<92:2963-2982>> <<type>> <<CLASS>> <<ANCHOR>>
<<__init__>> <<94:3013-3021>> <<function>> <<CONSTRUCTOR>> <<ANCHOR>>
<<ServerError>> <<99:3172-3183>> <<type>> <<CLASS>> <<ANCHOR>>
<<__init__>> <<101:3214-3222>> <<function>> <<CONSTRUCTOR>> <<ANCHOR>>
<<self>> <<61:1951-1954>> <<InvalidKeyError>> <<PARAMETER>> <<ANCHOR>>
<<key>> <<61:1957-1960>> <<?>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<61:1967-1970>> <<?>> <<PARAMETER>> <<ANCHOR>>
<<reason>> <<61:2023-2026>> <<str>> <<PARAMETER>> <<ANCHOR>>
<<key>> <<62:2050-2053>> <<?>> <<ATTRIBUTE>> <<ANCHOR>>
<<link>> <<63:2073-2077>> <<str>> <<ATTRIBUTE>> <<ANCHOR>>
<<message>> <<64:2118-2125>> <<?>> <<ATTRIBUTE>> <<ANCHOR>>
<<self>> <<17:560-563>> <<SafetyError>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<17:566-569>> <<str>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<18:622-629>> <<str>> <<ATTRIBUTE>> <<ANCHOR>>
<<self>> <<50:1617-1620>> <<DatabaseFileNotFoundError>> <<PARAMETER>> <<ANCHOR>>
<<db>> <<50:1623-1626>> <<?>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<50:1632-1635>> <<str>> <<PARAMETER>> <<ANCHOR>>
<<db>> <<51:1703-1705>> <<?>> <<ATTRIBUTE>> <<ANCHOR>>
<<message>> <<52:1724-1731>> <<str>> <<ATTRIBUTE>> <<ANCHOR>>
<<self>> <<87:2808-2811>> <<NetworkConnectionError>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<87:2814-2817>> <<str>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<88:2898-2905>> <<str>> <<ATTRIBUTE>> <<ANCHOR>>
<<self>> <<75:2465-2468>> <<TooManyRequestsError>> <<PARAMETER>> <<ANCHOR>>
<<reason>> <<75:2471-2474>> <<?>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<76:2501-2504>> <<str>> <<PARAMETER>> <<ANCHOR>>
<<self>> <<94:3022-3025>> <<RequestTimeoutError>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<94:3028-3031>> <<str>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<95:3107-3114>> <<str>> <<ATTRIBUTE>> <<ANCHOR>>
<<self>> <<81:2688-2691>> <<TooManyRequestsError>> <<PARAMETER>> <<ANCHOR>>
<<self>> <<44:1479-1482>> <<DatabaseFetchError>> <<PARAMETER>> <<ANCHOR>>
<<self>> <<7:295-298>> <<SafetyException>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<7:301-304>> <<str>> <<PARAMETER>> <<ANCHOR>>
<<info>> <<7:349-352>> <<str>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<8:372-379>> <<str>> <<ATTRIBUTE>> <<ANCHOR>>
<<self>> <<69:2349-2352>> <<InvalidKeyError>> <<PARAMETER>> <<ANCHOR>>
<<self>> <<34:1226-1229>> <<MalformedDatabase>> <<PARAMETER>> <<ANCHOR>>
<<self>> <<11:470-473>> <<SafetyException>> <<PARAMETER>> <<ANCHOR>>
<<self>> <<27:800-803>> <<MalformedDatabase>> <<PARAMETER>> <<ANCHOR>>
<<reason>> <<27:806-809>> <<?>> <<PARAMETER>> <<ANCHOR>>
<<fetched_from>> <<27:819-822>> <<str>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<28:859-862>> <<str>> <<PARAMETER>> <<ANCHOR>>
<<info>> <<30:1024-1028>> <<str>> <<VARIABLE>> <<ANCHOR>>
<<message>> <<31:1085-1092>> <<str>> <<ATTRIBUTE>> <<ANCHOR>>
<<self>> <<40:1330-1333>> <<DatabaseFetchError>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<40:1336-1339>> <<str>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<41:1399-1406>> <<str>> <<ATTRIBUTE>> <<ANCHOR>>
<<self>> <<21:702-705>> <<SafetyError>> <<PARAMETER>> <<ANCHOR>>
<<self>> <<55:1818-1821>> <<DatabaseFileNotFoundError>> <<PARAMETER>> <<ANCHOR>>
<<self>> <<101:3223-3226>> <<ServerError>> <<PARAMETER>> <<ANCHOR>>
<<reason>> <<101:3229-3232>> <<?>> <<PARAMETER>> <<ANCHOR>>
<<message>> <<102:3259-3262>> <<str>> <<PARAMETER>> <<ANCHOR>>
<<EXIT_CODE_FAILURE>> <<1:44-61>> <<int>> <<VARIABLE>> <<LINK>>
<<EXIT_CODE_INVALID_API_KEY>> <<1:63-88>> <<int>> <<VARIABLE>> <<LINK>>
<<EXIT_CODE_TOO_MANY_REQUESTS>> <<1:90-117>> <<int>> <<VARIABLE>> <<LINK>>
<<EXIT_CODE_UNABLE_TO_FETCH_VULNERABILITY_DB>> <<2:125-167>> <<int>> <<VARIABLE>> <<LINK>>
<<EXIT_CODE_UNABLE_TO_LOAD_LOCAL_VULNERABILITY_DB>> <<2:169-216>> <<int>> <<VARIABLE>> <<LINK>>
<<EXIT_CODE_MALFORMED_DB>> <<2:218-240>> <<int>> <<VARIABLE>> <<LINK>>
<<SafetyError>> <<25:768-779>> <<type>> <<VARIABLE>> <<LINK>>
<<SafetyError>> <<38:1298-1309>> <<type>> <<VARIABLE>> <<LINK>>
<<DatabaseFetchError>> <<48:1578-1596>> <<type>> <<VARIABLE>> <<LINK>>
<<DatabaseFetchError>> <<59:1912-1930>> <<type>> <<VARIABLE>> <<LINK>>
<<DatabaseFetchError>> <<73:2426-2444>> <<type>> <<VARIABLE>> <<LINK>>
<<DatabaseFetchError>> <<85:2769-2787>> <<type>> <<VARIABLE>> <<LINK>>
<<DatabaseFetchError>> <<92:2983-3001>> <<type>> <<VARIABLE>> <<LINK>>
<<DatabaseFetchError>> <<99:3184-3202>> <<type>> <<VARIABLE>> <<LINK>>
<<key>> <<62:2056-2059>> <<?>> <<VARIABLE>> <<LINK>>
<<self>> <<62:2045-2049>> <<InvalidKeyError>> <<VARIABLE>> <<LINK>>
<<self>> <<63:2068-2072>> <<InvalidKeyError>> <<VARIABLE>> <<LINK>>
<<key>> <<64:2171-2174>> <<?>> <<VARIABLE>> <<LINK>>
<<message>> <<64:2128-2135>> <<?>> <<VARIABLE>> <<LINK>>
<<key>> <<64:2147-2150>> <<?>> <<VARIABLE>> <<LINK>>
<<self>> <<64:2157-2161>> <<InvalidKeyError>> <<VARIABLE>> <<LINK>>
<<link>> <<64:2162-2166>> <<str>> <<VARIABLE>> <<LINK>>
<<message>> <<64:2180-2187>> <<?>> <<VARIABLE>> <<LINK>>
<<self>> <<64:2113-2117>> <<InvalidKeyError>> <<VARIABLE>> <<LINK>>
<<message>> <<18:632-639>> <<str>> <<VARIABLE>> <<LINK>>
<<self>> <<18:617-621>> <<SafetyError>> <<VARIABLE>> <<LINK>>
<<self>> <<19:665-669>> <<SafetyError>> <<VARIABLE>> <<LINK>>
<<message>> <<19:670-677>> <<str>> <<VARIABLE>> <<LINK>>
<<db>> <<51:1708-1710>> <<?>> <<VARIABLE>> <<LINK>>
<<self>> <<51:1698-1702>> <<DatabaseFileNotFoundError>> <<VARIABLE>> <<LINK>>
<<message>> <<52:1734-1741>> <<str>> <<VARIABLE>> <<LINK>>
<<db>> <<52:1752-1754>> <<?>> <<VARIABLE>> <<LINK>>
<<self>> <<52:1719-1723>> <<DatabaseFileNotFoundError>> <<VARIABLE>> <<LINK>>
<<self>> <<53:1781-1785>> <<DatabaseFileNotFoundError>> <<VARIABLE>> <<LINK>>
<<message>> <<53:1786-1793>> <<str>> <<VARIABLE>> <<LINK>>
<<message>> <<88:2908-2915>> <<str>> <<VARIABLE>> <<LINK>>
<<self>> <<88:2893-2897>> <<NetworkConnectionError>> <<VARIABLE>> <<LINK>>
<<self>> <<89:2941-2945>> <<NetworkConnectionError>> <<VARIABLE>> <<LINK>>
<<message>> <<89:2946-2953>> <<str>> <<VARIABLE>> <<LINK>>
<<message>> <<95:3117-3124>> <<str>> <<VARIABLE>> <<LINK>>
<<self>> <<95:3102-3106>> <<RequestTimeoutError>> <<VARIABLE>> <<LINK>>
<<self>> <<96:3150-3154>> <<RequestTimeoutError>> <<VARIABLE>> <<LINK>>
<<message>> <<96:3155-3162>> <<str>> <<VARIABLE>> <<LINK>>
<<EXIT_CODE_TOO_MANY_REQUESTS>> <<82:2710-2737>> <<int>> <<VARIABLE>> <<LINK>>
<<EXIT_CODE_UNABLE_TO_FETCH_VULNERABILITY_DB>> <<45:1501-1543>> <<int>> <<VARIABLE>> <<LINK>>
<<message>> <<8:382-389>> <<str>> <<VARIABLE>> <<LINK>>
<<info>> <<8:402-406>> <<str>> <<VARIABLE>> <<LINK>>
<<self>> <<8:367-371>> <<SafetyException>> <<VARIABLE>> <<LINK>>
<<self>> <<9:433-437>> <<SafetyException>> <<VARIABLE>> <<LINK>>
<<message>> <<9:438-445>> <<str>> <<VARIABLE>> <<LINK>>
<<EXIT_CODE_INVALID_API_KEY>> <<70:2371-2396>> <<int>> <<VARIABLE>> <<LINK>>
<<EXIT_CODE_MALFORMED_DB>> <<35:1248-1270>> <<int>> <<VARIABLE>> <<LINK>>
<<EXIT_CODE_FAILURE>> <<12:492-509>> <<int>> <<VARIABLE>> <<LINK>>
<<reason>> <<30:1064-1070>> <<?>> <<VARIABLE>> <<LINK>>
<<message>> <<31:1095-1102>> <<str>> <<VARIABLE>> <<LINK>>
<<fetched_from>> <<31:1123-1135>> <<str>> <<VARIABLE>> <<LINK>>
<<reason>> <<31:1148-1154>> <<?>> <<VARIABLE>> <<LINK>>
<<info>> <<31:1140-1144>> <<str>> <<VARIABLE>> <<LINK>>
<<self>> <<31:1080-1084>> <<MalformedDatabase>> <<VARIABLE>> <<LINK>>
<<self>> <<32:1189-1193>> <<MalformedDatabase>> <<VARIABLE>> <<LINK>>
<<message>> <<32:1194-1201>> <<str>> <<VARIABLE>> <<LINK>>
<<message>> <<41:1409-1416>> <<str>> <<VARIABLE>> <<LINK>>
<<self>> <<41:1394-1398>> <<DatabaseFetchError>> <<VARIABLE>> <<LINK>>
<<self>> <<42:1442-1446>> <<DatabaseFetchError>> <<VARIABLE>> <<LINK>>
<<message>> <<42:1447-1454>> <<str>> <<VARIABLE>> <<LINK>>
<<EXIT_CODE_FAILURE>> <<22:724-741>> <<int>> <<VARIABLE>> <<LINK>>
<<EXIT_CODE_UNABLE_TO_LOAD_LOCAL_VULNERABILITY_DB>> <<56:1840-1887>> <<int>> <<VARIABLE>> <<LINK>>
